src/librustc_mir/lib.rs

```diff
@@ -4,7 +4,6 @@ Rust MIR: a lowered representation of Rust. Also: an experiment!
 
 */
 
-#![feature(nll)]
```

Comment:
Can this be kept, it gives better error messages in some cases and ensures that we can't ignore future compatibility warnings.